train.py

```diff
@@ -242,7 +247,8 @@ def main():
                 writer.add_summary(summary, step)
 
             duration = time.time() - start_time
-            print('step %d - loss = %.3f, (%.3f sec/step)' % (step, loss_value, duration))
+            print('step %d - loss = %.3f, (%.3f sec/step)'
```

Comment:
I changed to use `.format()` instead of `%`, compared to the formatting in other lines.
But forgot to change format specifier...

I'll fix it soon.